harness/tests/common/untrusted-root/gen.sh

```diff
@@ -23,6 +23,7 @@ openssl req -new -config server.cnf -key "$key" -out "$csr"
 
 # Sign with rootca.
 echo "$rootca" | openssl x509 -req -days "$days" -sha512 -CAcreateserial \
+    -extensions req_v3_usr -extfile server.cnf \
```

Comment:
`server.cnf` is missing.

Reply:
I didn't have to change it, just needed to include it.